tests/Microsoft.Fx.Portability.MetadataReader.Tests/ManagedMetadataReaderTests.cs

```diff
@@ -245,5 +249,25 @@ public bool IsFrameworkAssembly(AssemblyReferenceInformation assembly)
                 return string.Equals(_assemblyName, assembly?.Name);
             }
         }
+
+        private class StreamFromFileAssemblyFile : IAssemblyFile
+        {
+            private readonly byte[] _bytes;
+            private readonly string _path;
+
+            public StreamFromFileAssemblyFile(string path)
+            {
+                _path = path;
+                _bytes = File.ReadAllBytes(path);
+            }
+
+            public bool Exists { get; } = true;
+
+            public string FullName => _path;
```

Comment:
I like 'Name' as the property name here; I agree that covers all the scenarios well.

Have we tested any scenarios where the name isn't a path?